src/main/scala/gitbucket/core/api/ApiTag.scala

```diff
@@ -0,0 +1,29 @@
+package gitbucket.core.api
+
+import gitbucket.core.util.RepositoryName
+
+case class ApiTagCommit(
+  sha: String,
+  url: ApiPath
+)
+
+case class ApiTag(
+  name: String,
+  commit: ApiTagCommit,
+  zipball_url: ApiPath,
+  tarball_url: ApiPath
+)
+
+object ApiTag {
+  def apply(
+    tagName: String,
+    repositoryName: RepositoryName,
+    commitId: String
+  ): ApiTag =
+    ApiTag(
+      name = tagName,
+      commit = ApiTagCommit(sha = commitId, url = ApiPath(s"/${repositoryName.fullName}/commits/${commitId}")),
+      zipball_url = ApiPath(s"/${repositoryName.fullName}/zipball/${tagName}"),
+      tarball_url = ApiPath(s"/${repositoryName.fullName}/tarball/${tagName}")
```

Comment:
Are these URLs actually available?

Reply:
@takezoe ah.. gitbucket doesn't have this urls. Ok I will make them disable!

Reply:
I think we can use the following URLs:
- `/${repositoryName.fullName}/archive/${tagName}.zip`
- `/${repositoryName.fullName}/archive/${tagName}.tar.gz`

Reply:
@takezoe Thanks for the urls! I fixed PR.